etc/fiberassign_compile.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#
+# This script is intended to be run by desiInstall and only when installing
+# branches. However, it can also be run by hand.
+#
+# The single command-line argument is the path to the Python executable
+# that is calling desiInstall.
+#
+# This is meant to be run in ${INSTALL_DIR}, which is automatically
+# defined by desiInstall.
+#
+# As of 2022, setup.py in generally deprecated, so whatever replaces it
+# can go in this script.
+#
+py=$1
+cd ${INSTALL_DIR}
```

Comment:
I have tested this with desihub/desiutil#188 and it works fine, but see the suggestion there to put the responsibility on desiInstall to cd to the top level directory first before calling this script instead of setting an environment variable that the scripts have to know about to be able to find that top-level directory.